Visual Studio Build Error

Just hope that maybe somebody else came across the same error: 
```
error MSB4062: The "Bootsharp.Publish.BootsharpEmit" task could not be loaded from the assembly C:\Users\dnhb\.nuget\packages\bootsharp\0.1.3\build\..\tasks\Bootsharp.Publish.dll. Could not load file or assembly 'System.Runtime, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.
4>Done building target "BootsharpEmit" in project "SVN.MacTool.Validation.csproj" -- FAILED
```

`dotnet build` is running just fine. Only VS gives problems.

--- a/src/cs/Bootsharp/Build/Bootsharp.targets
+++ b/src/cs/Bootsharp/Build/Bootsharp.targets
@@ -39,8 +39,18 @@
         <EmscriptenEnvVars Include="EM_CACHE=$(EmscriptenCacheSdkCacheDir)"/>
     </ItemGroup>
 
-    <UsingTask TaskName="Bootsharp.Publish.BootsharpEmit" AssemblyFile="$(BootsharpPublishAssembly)"/>
-    <UsingTask TaskName="Bootsharp.Publish.BootsharpPack" AssemblyFile="$(BootsharpPublishAssembly)"/>
+    <!-- Register tasks for Visual Studio. (https://github.com/elringus/bootsharp/issues/137) -->
+    <UsingTask Condition="'$(MSBuildRuntimeType)' == 'Full'"
+               TaskName="Bootsharp.Publish.BootsharpEmit" AssemblyFile="$(BootsharpPublishAssembly)"
+               Runtime="NET" TaskFactory="TaskHostFactory"/>
+    <UsingTask Condition="'$(MSBuildRuntimeType)' == 'Full'"
+               TaskName="Bootsharp.Publish.BootsharpPack" AssemblyFile="$(BootsharpPublishAssembly)"
+               Runtime="NET" TaskFactory="TaskHostFactory"/>
+    <!-- Register tasks for 'dotnet publish' run from CLI. -->
+    <UsingTask Condition="'$(MSBuildRuntimeType)' != 'Full'"
+               TaskName="Bootsharp.Publish.BootsharpEmit" AssemblyFile="$(BootsharpPublishAssembly)"/>
+    <UsingTask Condition="'$(MSBuildRuntimeType)' != 'Full'"
+               TaskName="Bootsharp.Publish.BootsharpPack" AssemblyFile="$(BootsharpPublishAssembly)"/>
 
     <!-- A hack due to source generator compositing not possible. (https://github.com/dotnet/roslyn/issues/57239#issuecomment-1585235740) -->
     <Target Name="BootsharpEmit" BeforeTargets="GenerateAdditionalSources"

--- a/src/cs/Directory.Build.props
+++ b/src/cs/Directory.Build.props
@@ -1,7 +1,7 @@
 <Project>
 
     <PropertyGroup>
-        <Version>0.8.0-alpha.44</Version>
+        <Version>0.8.0-alpha.45</Version>
         <Authors>Elringus</Authors>
         <PackageTags>javascript typescript ts js wasm node deno bun interop codegen</PackageTags>
         <PackageProjectUrl>https://bootsharp.com</PackageProjectUrl>

